frontend/src/components/@common/Searchbar/SearchbarSelectBox.tsx

```diff
@@ -0,0 +1,63 @@
+import { forwardRef } from 'react';
+import styled from 'styled-components';
+import { Option } from '~/@types/utils.types';
+
+interface SearchbarSelectBoxProps {
+  options: Option[];
+  onClickEvent: (option: Option) => () => void;
+}
+
+const SearchbarSelectBox = forwardRef<HTMLUListElement, SearchbarSelectBoxProps>(({ options, onClickEvent }, ref) => (
+  <SearchbarTags ref={ref}>
+    {options.map(option => (
+      <SearchBarTag key={option.key} value={option.value} onClick={onClickEvent(option)}>
+        {option.value}
+      </SearchBarTag>
+    ))}
+  </SearchbarTags>
+));
+
+export default SearchbarSelectBox;
+
+const SearchbarTags = styled.ul`
+  border-radius: 16px;
+
+  width: 741px;
+
+  padding: 0 2.1rem 0 2.1rem;
+
+  font-size: 1.6rem;
```

Comment:
`styles/common` 에 적용해 놓은 상수값으로 작성해 주세요!

```suggestion
  font-size: ${FONT_SIZE.sm}; // 이런식으로
```

Reply:
> 수정했습니다!! 👍👍